src/interfaces/options.ts

```diff
@@ -0,0 +1,9 @@
+import RedisStorage from "../storages/redis-storage";
```

Comment:
```suggestion
import RedisStorage from "@src/storages/redis-storage";
```
We need to use absolute imports across modules, and relative imports within module. Need to set it up

Reply:
Just adding it in TODO for now as base setup is needed for progressing with other strategies.

Reply:
Done